app/controllers/merchant/items_controller.rb

```diff
@@ -5,6 +5,16 @@ def index
     @items = current_user.merchant.items
   end
 
+  def new
+  end
+
+  def create
+    merchant = current_user.merchant
+    merchant.items.create!(item_params)
```

Comment:
I need to remember to utilize the bang on the create method to make sure it's created correctly

Reply:
second that!!